processor/src/host/advice/map.rs

```diff
@@ -0,0 +1,43 @@
+use super::{BTreeMap, Felt, Vec};
+
+extern crate alloc;
+use alloc::collections::btree_map::IntoIter;
+
+use vm_core::crypto::hash::RpoDigest;
+
+// ADVICE MAP
+#[derive(Debug, Clone, Default)]
+pub struct AdviceMap(BTreeMap<RpoDigest, Vec<Felt>>);
```

Comment:
Would be great to add some basic doc comments for the struct as well as its public functions.